### PR TITLE
Restored Example 9 and added CVM as Example 13.

### PR DESCRIPTION
**Pull request by AjKundnani**
## Description

Related to #1222, example 9 was replaced with CVM content. Restored original Example 9 and added new Example 13 for Confidential VM support.
## Checklist

- [x] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

**Comment by grizzlytheodore** [comment]
@Nickcandy please review, does the static analysis need to be suppressed or can we merge still?

File: src/Compute/Compute/help/New-AzGalleryImageVersion.md [hunk 1]
```diff
@@ -235,6 +233,33 @@ New-AzGalleryImageVersion -ResourceGroupName $rgName -GalleryName $galleryName -
...
+$galleryName = "myGallery"
+$galleryImageDefinitionName = "cvmImage"
+$galleryImageVersionName = "1.0.0"
+$location = "North Europe"
+
+$sourceImageId = "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myVMRG/providers/Microsoft.Compute/virtualMachines/myVM"
+$cvmDiskEncryptionSetId = "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myVMRG/providers/Microsoft.Compute/diskEncryptionSets/cvmDiskEncryptionSet"
+$dataDiskEncryptionSetId = "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myVMRG/providers/Microsoft.Compute/diskEncryptionSets/dataDiskEncryptionSet"
+
+$cvmOsDiskEncryption = @{CVMEncryptionType=EncryptedWithCmk; CVMDiskEncryptionSetID=$cvmDiskEncryptionSetId}
```
[the remark this answers is not in the document]
Comment (grizzlytheodore) [reply src/Compute/Compute/help/New-AzGalleryImageVersion.md:249]: that is string. @AjKundnani  could you put it around quotations?

[the remark this answers is not in the document]
Comment (AjKundnani) [reply src/Compute/Compute/help/New-AzGalleryImageVersion.md]: It is a string, thanks.

**Review by Nickcandy — APPROVED** [review]

**Pull request merged**